hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/WritableNftStore.java

```diff
@@ -101,4 +101,26 @@ public long sizeOfState() {
     public Set<UniqueTokenId> modifiedNfts() {
         return nftState.modifiedKeys();
     }
+
+    /**
+     * Removes the {@link Nft} with the given serial number
+     *
+     * @param serialNum - the combined unique ID of the NFT to remove
+     */
+    public void remove(final @NonNull UniqueTokenId serialNum) {
+        nftState.remove(requireNonNull(serialNum));
+    }
+
+    /**
+     * Removes the {@link Nft} with the given serial number
+     *
+     * @param tokenId - the token id of the NFT to remove
+     * @param serialNum - the serial number of the NFT to remove
+     */
+    public void remove(final @NonNull TokenID tokenId, final long serialNum) {
+        remove(UniqueTokenId.newBuilder()
+                .tokenTypeNumber(tokenId.tokenNum())
+                .serialNumber(serialNum)
+                .build());
```

Comment:
I would have used the `copyBuilder` here, unless there is a reason not to.

Reply:
I guess I'm confused. We're given a `TokenID` object, but not a `UniqueTokenId` object. What would we copy here?